### PR TITLE
Adding templates for github

### PR DESCRIPTION
**Pull request by eggerdj**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.black]
+line-length = 100
+target-version = ['py36', 'py37', 'py38']
```
Comment (LaurinFischer) [inline pyproject.toml:6]: Any particular reason why `py39` is not (yet) included?

File: tools/verify_headers.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020
+#
```
Comment (LaurinFischer) [inline tools/verify_headers.py:5]: not sure if this should be updated to 2021

File: qiskit_cold_atom/applications/fermi_hubbard.py [hunk 3]
```diff
@@ -89,9 +89,7 @@ def __init__(
         self.particles_down = particles_down
         self.J = hop_strength
         self.U = int_strength
-        self.basis = FermionicBasis(
-            self.size, n_particles=[self.particles_up, self.particles_down]
-        )
+        self.basis = FermionicBasis(self.size, n_particles=[self.particles_up, self.particles_down])
```
Comment (LaurinFischer) [inline qiskit_cold_atom/applications/fermi_hubbard.py:92]: black used to force these linebreaks, so I'm wondering why this is now being changed? Has something changed in the black setup/preferences? (Is there even such a thing?)

Comment (eggerdj) [reply qiskit_cold_atom/applications/fermi_hubbard.py:92]: This depends on the line length. The checks that we have are taken from qiskit-experiments so we are consistent with other packages.

File: tools/verify_headers.py (hunk 2 above)
Comment (eggerdj) [reply tools/verify_headers.py]: Changed here 614dad7.

File: pyproject.toml (hunk 1 above)
Comment (eggerdj) [reply pyproject.toml]: Thanks, probably ok to add. See here: 6976ad4

**Review by LaurinFischer — APPROVED** [review]
LGTM

**Pull request merged**